package.json

```diff
@@ -41,7 +42,8 @@
     "slick-carousel": "1.8.1",
     "react-svg": "14.0.4",
     "@eeacms/volto-arcgis-block": "*",
-    "react-native-mime-types": "2.3.0"
+    "react-native-mime-types": "2.3.0",
+    "@eeacms/volto-clms-utils":"git+http://github.com/eea/volto-clms-utils.git#develop"
```

Comment:
We should release an initial version of volto-clms-utils and then depend on a released version instead of a git version

Reply:
Comments @ionlizarazu ?